kong/clustering/data_plane.lua

```diff
@@ -262,11 +265,20 @@ function _M:communicate(premature)
   ngx.thread.kill(write_thread)
   c:close()
 
+  local err_msg
```

Comment:
You can just write `local err_msg = ok and err or perr` instead. Might be easier to read.